_posts/2020-03-11-faf-2-2-0.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "FAF 2.2.0 released"
+date:   2020-03-11 14:00:00 +0100
+author: Michal Fabík
+visible: 1
+categories: release features faf "ABRT Analytics"
+---
+
+A new version of FAF has just been released. From the user's point of view, you will probably find the following changes the most noticable:
+
++ A markedly sped up `create-problems` action.
++ A more capable CLI with autocompletion and wildcard support
++ Many web UI improvements. <sub>(Still evaluating some changes to the table layout though.)</sub>
```

Comment:
I would drop the `Still evaluating some changes to the table layout though.` part.

Reply:
Dropped.